src/test/java/org/inferred/freebuilder/processor/MapSourceTest.java

```diff
@@ -62,29 +62,25 @@ public void test_guava_j6() {
         "  /**",
         "   * Associates {@code key} with {@code value} in the map to be returned from",
         "   * {@link Person#getName()}.",
-        "   * Duplicate keys are not allowed.",
+        "   * If the map previously contained a mapping for the key,",
+        "   * the old value is replaced by the specified value.",
         "   *",
         "   * @return this {@code Builder} object",
         "   * @throws NullPointerException if {@code value} is null",
-        "   * @throws IllegalArgumentException if {@code key} is already present",
         "   */",
         "  public Person.Builder putName(int key, String value) {",
         "    Preconditions.checkNotNull(value);",
-        "    Preconditions.checkArgument(!name.containsKey(key), "
-            + "\"Key already present in name: %s\", key);",
         "    name.put(key, value);",
         "    return (Person.Builder) this;",
         "  }",
         "",
         "  /**",
-        "   * Associates all of {@code map}'s keys and values in the map to be returned",
-        "   * from {@link Person#getName()}.",
-        "   * Duplicate keys are not allowed.",
+        "   * Copies all of the mappings from {@code map} to the map to be returned from",
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The String literal "   * Copies all of the mappings from {@code map} to the map to be returned from" appears 4 times in this file; the first occurrence is on line 78](https://www.codacy.com/app/Chris-Purcell-39/FreeBuilder/file/2409332806/issues/source?bid=3146243&fileBranchId=3234738#l78)